pkg/logs/config/integration_config_test.go

```diff
@@ -26,8 +26,8 @@ func TestBuildLogsAgentIntegrationsConfigs(t *testing.T) {
 	allSources, err := buildLogSources(ddconfdPath, false)
 
 	assert.Nil(t, err)
-	assert.Equal(t, 7, len(allSources.GetValidSources()))
-	assert.Equal(t, 8, len(allSources.GetSources()))
+	assert.Equal(t, 8, len(allSources.GetValidSources()))
+	assert.Equal(t, 9, len(allSources.GetSources()))
```

Comment:
could we do `1, len(allSources.GetSources()) - len(allSources.GetValidSources())` to avoid always updating these two lines?